src/build/bnd/scala-compiler.bnd

```diff
@@ -5,7 +5,6 @@ Bundle-Version: ${ver}
 Export-Package: *;version=${ver}
 Import-Package: jline.*;resolution:=optional, \
                 org.apache.tools.ant.*;resolution:=optional, \
-                scala.util.parsing.*;version="${range;[====,====];@PARSER_COMBINATORS_VERSION@}";resolution:=optional, \
```

Comment:
There's a similar import spec in `build.sbt` that needs to be changed. This file is only used by ant.